validation.js

```diff
@@ -19,3 +20,16 @@ form.addEventListener('submit', (event) => {
   }
   event.preventDefault();
 });
+
+function saveInfo() {
+  const userInfo = JSON.stringify({ username: userName.value, email: email.value });
+  localStorage.setItem('userInfo', userInfo);
+}
```

Comment:
Good job on persisting your data to localStorage, but, you are missing one more field;

- [ ] Kindly add the message input field as well so that one too is stored in localStorage just as the name and email input fields. 👍🏽 

<img width="800" alt="Screenshot 2023-03-11 at 1 25 53 AM" src="https://user-images.githubusercontent.com/50941074/224457511-4d04b40a-8aa9-4d9a-bae5-9f12e037ed2a.png">